meta-sokol-flex-distro/conf/distro/sokol-flex.conf

```diff
@@ -72,7 +72,7 @@ SANITY_TESTED_DISTROS = "\
 
 # Sane default append for the kernel cmdline (not used by all BSPs)
 # TODO: rename APPEND to something more appropriate
-APPEND ?= "${APPEND_SERIAL} console=tty0 ${APPEND_SPLASH}"
+APPEND ?= "${APPEND_SERIAL} ${APPEND_SPLASH} ${mtdargs} ${APPEND_GRUB_ROOTARGS}"
```

Comment:
but there are no grub specific bootargs?

Reply:
Why not move all of this to flex-bsp-common.inc? And like we discussed on the chat, nothings grub specific now so you may want to rename this variable (if this is at all needed)